.github/workflows/tests.jsonnet

```diff
@@ -494,6 +494,26 @@ local push_docker_nonroot_job = {
   },
 };
 
+local build_test_docker_performance_tests_job = build_test_docker_nonroot_job + {
+  with: super.with + {
+    'docker-flavor': |||
+      latest=auto
+      suffix=-performance-tests,onlatest=true
+    |||,
+    'artifact-name': 'image-test-performance-tests',
+    target: 'performance-tests',
+  },
+};
+
+local push_docker_performance_tests_job = push_docker_nonroot_job + {
```

Comment:
Checking my understanding: this just runs `build-test-docker-performance-tests-job` and uploads the same artifact? (while doing the other push_docker workflow)

Reply:
The `nonroot` build handles everything I want this job to do already, so this uses the nonroot job with some modifications.
This jsonnet eventually renders out the YAML in `tests.yml`.
The `build-test-docker-performance-tests-job` builds the container image specified by the `performances-tests` build target in the `Dockerfile`, and the `push...` job pushes that image to Dockerhub.